### ISSUE TITLE
Change Year Directly from control

### ISSUE THREAD

**Issue by tpruthi** [jkuri/ng2-datepicker#52]
Is there any way to change to year directly like previous version

**Comment by jkuri** [comment]
unfortunately not. it was nasty looking with that double arrows and I decided to remove it with the new version.

**Comment by tpruthi** [comment]
But is was a good feature. If end user will have to select DOB as 17-04-1952. He will have to press a lot of time. Which is not good

**Comment by jkuri** [comment]
you are right about that for sure. its not a feature that's hard to implement just didn't found the way to be consistent with the new design. is the double click maybe an option?

**Comment by tpruthi** [comment]
For a better design you can implement like windows datepicker. where on clicking the month no of months appear and on clicking again on year, It populates a list of years. It is also implemented in ng2-bootstrap

**Comment by RajniShriramoji** [comment]
Hi, can we use old version for changing year directly as this is removed in latest version.
Can you please suggest which version we need to use for changing year directly.

Thanks!

**Comment by hydra1983** [comment]
Maybe ```allowToChangeYears``` can be provided to achieve this.

```typescript
    prevYear() {
        this.date = this.date.subtract(1, 'year');
        this.generateCalendar();
    }

    nextYear() {
        this.date = this.date.add(1, 'year');
        this.generateCalendar();
    }
```
